docker-compose.yml

```diff
@@ -1,18 +1,36 @@
 version: "3"
 services:
-  website:
-    image: openstates/new-openstates.org
-    volumes:
-      - .:/opt/openstates/openstates.org/
-    entrypoint: ['/bin/bash']
-
   database:
     # With this service up, your host can use a database URL of
     # `postgres://openstates@localhost:5433/openstates`
-    image: mdillon/postgis:9.6
+    image: mdillon/postgis:9.4-alpine
```

Comment:
why not pg 9.6?

Reply:
This is the version that is mentioned in readme. So I assumed this is what is being used on the server.